
FastLoose:
  acc_lim_x: 3.5
  acc_lim_y: 0.0
  acc_lim_theta: 0.5
  #acc_lim_trans:  = acc_lim_x
  max_vel_trans: 0.45
  min_vel_trans: 0.1
  max_vel_x: 0.45
  min_vel_x: -0.1 #!!
  max_vel_y: 0.0
  min_vel_y: 0.0
  max_vel_theta: 0.5
  min_vel_theta: 0.1

  trans_stopped_vel: 0.1
  theta_stopped_vel: 0.1

  yaw_goal_tolerance: 0.15
  xy_goal_tolerance: 0.5
  trans_stopped_vel: 0.1
  theta_stopped_vel: 0.1

Default:
  #restore_defaults: false #default: false
  acc_lim_x: 1.0 #2.5
  acc_lim_y: 0.0 #2.5
  acc_lim_theta: 0.4 #3.2
  #acc_lim_trans: 0.1 #1.0
  max_vel_trans: 0.30 #0.55
  min_vel_trans: 0.00 #0.1
  max_vel_x: 0.3 #0.55
  min_vel_x: 0.0
  max_vel_y: 0.0 #0.1
  min_vel_y: 0.0 #-0.1
  max_vel_theta: 0.4 #1.0
  min_vel_theta: 0.0 #0.4

  trans_stopped_vel: 0.1
  theta_stopped_vel: 0.1

  yaw_goal_tolerance: 0.08 #0.05
  xy_goal_tolerance: 0.20 #0.1
